exchanges/coinbasepro/coinbasepro_wrapper.go

```diff
@@ -832,17 +829,22 @@ func (c *CoinbasePro) GetOrderHistory(ctx context.Context, req *order.GetOrdersR
 		orderSide := order.Side(strings.ToUpper(respOrders[i].Side))
 		orderType := order.Type(strings.ToUpper(respOrders[i].Type))
 		orders = append(orders, order.Detail{
-			ID:             respOrders[i].ID,
-			Amount:         respOrders[i].Size,
-			ExecutedAmount: respOrders[i].FilledSize,
-			Type:           orderType,
-			Date:           respOrders[i].CreatedAt,
-			Fee:            respOrders[i].FillFees,
-			FeeAsset:       curr.Quote,
-			Side:           orderSide,
-			Pair:           curr,
-			Price:          respOrders[i].Price,
-			Exchange:       c.Name,
+			ID:                   respOrders[i].ID,
+			Amount:               respOrders[i].Size,
+			ExecutedAmount:       respOrders[i].FilledSize,
+			RemainingAmount:      respOrders[i].Size - respOrders[i].FilledSize,
+			Cost:                 respOrders[i].ExecutedValue,
```

Comment:
Reference: https://docs.cloud.coinbase.com/exchange/reference/exchangerestapi_getorders

"executed_value is the cumulative match size * price and is only present for orders placed after 2016-05-20."